### PR TITLE
Rooms

### PR DESCRIPTION
**Pull request by muzammildafedar**
Add, Delete and get all the rooms apis integrated.

### CODE REVIEW

File: src/components/admin/rooms/Room.reducer.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  const initialState = Object.freeze({
+    result: [],
+    requestStatus: null,
+    requestError: null,
+    getUserRes: [],
+    getUserStatus: null,
+    getUserError: null,
+  });
+  
+  export default function userReducer(state = initialState, action) {
```
Comment (vishwanath-excellarate) [inline src/components/admin/rooms/Room.reducer.js]: Update the function name

File: src/components/admin/rooms/Room.reducer.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    GET_ROOM_REQUEST,
+    GET_ROOM_SUCCESS,
+    GET_ROOM_FAILURE,
+  } from "./Room.action.constants";
+  
+  const initialState = Object.freeze({
+    result: [],
+    requestStatus: null,
+    requestError: null,
+    getUserRes: [],
```
Comment (vishwanath-excellarate) [inline src/components/admin/rooms/Room.reducer.js]: modify the variable names

Comment (muzammildafedar) [reply src/components/admin/rooms/Room.reducer.js]: [Delete issue fixed](https://github.com/vishwanath-somavarad/guest-house-portal/pull/2/commits/bb5700bf23d8b4dc294076a6f081828e273520fc)

already fixed in this commit

File: src/components/admin/rooms/Room.reducer.js (hunk 1 above)
Comment (muzammildafedar) [reply src/components/admin/rooms/Room.reducer.js]: [Delete issue fixed](https://github.com/vishwanath-somavarad/guest-house-portal/pull/2/commits/bb5700bf23d8b4dc294076a6f081828e273520fc)

already fixed in this commit

**Pull request merged**